worksheet_A/TerminalHacking.cpp

```diff
@@ -40,7 +54,45 @@ int main()
 	}
 
 	// TODO: implement the rest of the game
+	int Likeness = 0;
+	int IncorrectGuesses = 0;
+	bool inlist;
+	std::string guess;
+	
+	while (IncorrectGuesses < 4)
+	{
+		inlist = false;
+		while (inlist == false)
+		{
+			std::cout << "Type word in UPPERCASE" << std::endl;
+			std::cin >> guess;
 
+			// loops through all the words in the word list
+			for each (std::string word in options)
```

Comment:
This is fine, but there is an easier (and more computationally efficient) way to check if a `std::set` contains an element -- see http://stackoverflow.com/questions/1701067/how-to-check-that-an-element-is-in-a-stdset